docs-dev/DEVELOPMENT_LOCAL.md

```diff
@@ -0,0 +1,117 @@
+# Local Development and Debug Environments
+
+This document is a collection of resources that may be helpful to new ceramic developers.
+
+## IDE configuration
+
+### WebStorm
+
+Open the js-ceramic project.  Set the Run/Debug configuration template (under Run..Edit Configuration..Edit Configuration Templates) as follows:
+
+![edit configuration templates](https://user-images.githubusercontent.com/798887/169563176-f6e15e71-8bf3-4f7f-a5d4-ce90732067e1.png)
+
+
+Note this uses the dev-unstable network, it is also possible to configure for clay testnet.  The Application parameters used in this example are
+
+```
+daemon --port 7007 --hostname 0.0.0.0 --network dev-unstable --anchor-service-api https://cas-dev.3boxlabs.com --debug true --ethereum-rpc https://rinkeby.infura.io/v3/b6685df41e1647c4be0046dfa62a020b
+```
+This configuration will allow running the ceramic daemon from the debugger and setting breakpoints in the daemon.
+
+
+## Local Development
+
+*macOS pre-requisits (11.x+):*
+
+* Install XCode & Command Line tools: `xcode-select --install`
+
+* Install Brew: https://brew.sh
+
+`brew install pyenv`
+
+Add pyenv to your PATH so that you can reference python (not python3):
```

Comment:
Consider stating which versions of Python are needed, rather than saying `not python3`.

I'm also struggling to see why I need a Python Environment.  Is ti needed for WebStorm to run properly?  What is the purpose of this step?

Reply:
I am not clear on it - @willex ?

Reply:
Yeah, going to update it. By default there is no `python` alias available when using a vanilla MacOS install which makes the standard npm install fail (just ran into that issue with Artur setting up his new laptop).